corehq/apps/app_manager/views/media_utils.py

```diff
@@ -26,10 +26,15 @@ def interpolate_media_path(val):
     return val
 
 
-def handle_media_edits(request, item, should_edit, resp, lang):
+def handle_media_edits(request, item, should_edit, resp, lang, app):
     if 'corrections' not in resp:
         resp['corrections'] = {}
     for attribute in ('media_image', 'media_audio'):
         if should_edit(attribute):
+            old_value = getattr(item, attribute, {})[lang]
```

Comment:
if `attribute` doesn't exist, this'll be `{}[lang]`, which will raise.  Should it be `.get(lang, None)`?

Reply:
Aha! thanks @esoergel